docs/webhooks.rst

```diff
@@ -40,6 +40,17 @@ your docs whenever you push updates:
 If you ever need to manually set the webhook on Bitbucket,
 you can point it at ``https://readthedocs.org/bitbucket``.
 
+GitLab
+------
+
+If your project is hosted on GitLab, you can manually set the webhook on Gitlab and
+point it at ``https://readthedocs.org/gitlab``:
```

Comment:
Should this webhook be updated to reflect the new webhook end points added in #2433 ?